README.md

```diff
@@ -1361,6 +1361,7 @@ _See also: [awesome-sysadmin/IT Asset Management](https://github.com/awesome-fos
 - [Invoice Ninja](https://www.invoiceninja.org/) - Powerful tool to invoice clients online. ([Demo](https://app.invoiceninja.com/invoices/create), [Source Code](https://github.com/invoiceninja/invoiceninja)) `AAL` `PHP`
 - [InvoicePlane](https://github.com/InvoicePlane/InvoicePlane) - Manage quotes, invoices, payments and customers for your small business. `MIT` `PHP`
 - [Kresus](https://kresus.org/) - Open source personal finance manager. ([Demo](https://kresus.org/en/demo.html), [Source Code](https://github.com/kresusapp/kresus)) `MIT` `Nodejs`
+- [Lago](https://www.getlago.com/) - Open-source metering and usage-based billing. ([Source Code](https://github.com/getlago/lago)) `AGPL-3.0` `Shell`
```

Comment:
```suggestion
- [Lago](https://www.getlago.com/) - Open-source metering and usage-based billing. ([Source Code](https://github.com/getlago/lago)) `AGPL-3.0` `Docker`
```

> Requirements
> Install Docker on your machine;

> The Language tag is the main server-side requirement for the software

Reply:
So, requirements includes a platform supporting Docker? Slackware is out.

Reply:
@trasherdk we provide native support for Docker so that the easiest way to get started with Lago, it's still possible to run Lago by cloning the repo and doing the conf on any other platform!

Reply:
Docker is not mandatory but we provide a support for it, I propose an alternative solution if that suits you best:
```suggestion
- [Lago](https://www.getlago.com/) - Open-source metering and usage-based billing. ([Source Code](https://github.com/getlago/lago)) `AGPL-3.0` `Ruby/React`
```

Reply:
What matters is which installation methods are documented. Since only docker-based installations are documented [[1]](https://github.com/getlago/lago#-deploy-locally), the language tag must be `Docker`, unless someone adds documentation for other methods.